lib/breakers/statsd_plugin.rb

```diff
@@ -2,12 +2,15 @@
 
 module Breakers
   class StatsdPlugin
-    def get_tags(request)
+    def get_tags(request, response = nil)
       tags = []
       if request
         tags.append("endpoint:#{filtered_endpoint_tag(request.url.path)}") if request.url&.path
         tags.append("method:#{request.method}") if request.method
       end
+      if response
```

Comment:
I guess response could be nil if there was an exception. I'm curious if this http status would be captured before any other middleware would have possibly updated the http status code or raises an error. When would response be nil? In the event of a 404?

Reply:
Those are good questions.  I'd need to step through code to really answer that.